lib/opentok/token_generator.rb

```diff
@@ -81,6 +81,15 @@ def generate_token
         end
         data_params[:connection_data] = data
       end
+
+      if opts[:initial_layout_classes]
```

Comment:
@jeffswartz thanks for the clarification. @aiham looks like the data type should be an array of string.